README.md

```diff
@@ -337,3 +337,54 @@ NAMESPACE_CLEANUP
 HTTP_STRESS_CONTAINER_IMAGE
 HTTP_SERVER_CONTAINER_IMAGE
 ```
+
+### PVC Scale Workload 
+
+This test creates desired number of pods with PVC attached and them measure 
+time necessary for pods to start. The focus is on storage backend scalability.
+One can create different scenarios with various PVC sizes and them compare time 
+necessary for pods to start with various storage backends. 
+
+Prerequesties : 
+
+PVC scale test expects fully functional storage backend which can be used to
+dynamically allocate PVC, in this test storage/storage class is not configured. 
+If storage backend is not functional and dynamic PVC allocation is not working 
+then this test will not work. 
+
```

Comment:
It might seem silly but I think we should include just the short four steps or so to run the playbook something along the lines of

```sh
$ cp workloads/inventory.example inventory
$ # Add orchestration host to inventory
$ # Edit vars in workloads/vars/pvcscale.yml or define Environment vars (See below)
$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
```